hack/api-reference/core.md

```diff
@@ -9316,7 +9316,7 @@ string
 <a href="#core.gardener.cloud/v1beta1.ExpirableVersion">ExpirableVersion</a>)
 </p>
 <p>
-<p>VersionClassification is the logical state of a version according to <a href="https://github.com/gardener/gardener/blob/master/docs/operations/versioning.md">https://github.com/gardener/gardener/blob/master/docs/operations/versioning.md</a></p>
+<p>VersionClassification is the logical state of a version according to <a href="https://github.com/gardener/gardener/blob/master/docs/operations/versioning.md">https://github.com/gardener/gardener/blob/master/docs/proposals/05-versioning-policy.md</a></p>
```

Comment:
Instead of updating this link, I would rather vote to remove it from the doc string. The doc string should be good enough even without it.